src/settings/RequestPolicy/RequestPolicyForm.js

```diff
@@ -68,10 +74,25 @@ class RequestPolicyForm extends React.Component {
     this.props.onSave({ ...data, requestTypes });
   };
 
+  isDeletionALlowed() {
+    const { parentResources, policy } = this.props;
+    const circulationRules = get(parentResources, 'circulationRules.records[0].rulesAsText', '');
+
+    return !circulationRules.match(policy.id);
+  }
+
   changeDeleteState = (showDeleteConfirmation) => {
-    this.setState({ showDeleteConfirmation });
+    if (showDeleteConfirmation && !this.isDeletionALlowed()) {
```

Comment:
Odd casing on isDeletionALlowed ... might want to fix that.

Reply:
Nice catch @Baroquem. Thank you. Will fix it soon.